tracing-subscriber/src/registry/mod.rs

```diff
@@ -261,20 +424,14 @@ where
     /// next span, and then the next, until this span's immediate parent is
     /// returned.
     ///
-    /// **Note**: if the "smallvec" feature flag is not enabled, this may
-    /// allocate.
+    /// **Note**: this will allocate if there are many spans remaining, or if the
+    /// "smallvec" feature flag is not enabled.
+    #[deprecated(
+        note = "equivalent to `self.parent().into_iter().flat_map(|span| span.scope().from_root())`, but consider whether excluding self is actually intended"
```

Comment:
```suggestion
        note = "equivalent to `self.parent().into_iter().flat_map(|span| span.scope().from_root())`, but consider whether excluding `self` is actually intended",
        since = "0.2.19",
```